conf/messages

```diff
@@ -343,8 +343,23 @@ updateEmail.p3.list1 = updates on changes to the Customs Declaration Service
 updateEmail.p3.list2 = urgent updates about goods in customs
 updateEmail.p3.list3 = some financial notifications, including Direct Debit notices and VAT
 
-become-admin.title=[TODO - become admin title]
-become-admin.hint=[TODO - become admin hint]
-become-admin-confirmation.title = You have requested to become the administrator for this single undertaking
 
-become-admin-terms-and-conditions.title=[TODO - become admin terms and conditions title]
+become-admin.title=Do you want to become the administrator for {0}
+become-admin.hint=As undertaking administrator, the business EORI number {0} will be responsible for reporting de minimis aid (less Customs Duty waivers) payments received by all businesses which are part of this single undertaking.
+
+become-admin-tandc.title=You must accept the terms and conditions
+become-admin-tandc.h2=Reporting de minimis aid payments (less Customs Duty waivers) as a single undertaking
+become-admin-tandc.p1=De minimis aid payments (less Customs Duty waivers) could include subsidised contracts, loans or grants, from organisations like Invest NI and NI Direct. If you are unsure if an aid you claim is counted as de minimis aid payments (less Customs Duty waivers) you should contact the provider.
+become-admin-tandc.h2-2=What is a single undertaking?
+become-admin-tandc.p2=HMRC needs to group together related businesses within a company or organisation into a ‘single undertaking’. Businesses within an undertaking are identified by their EORI number.
```

Comment:
quotes around ‘single undertaking’ looks weird . I think they should be like 'single undertaking'

Reply:
It's a requirement to be smart quotes. Also the messages file ignores single quotes, which Could be escaped, but it's still a requirement to use smart quotes